crates/ui/src/components/title_bar/windows_window_controls.rs

```diff
@@ -110,6 +108,8 @@ impl RenderOnce for WindowsCaptionButton {
             .content_center()
             .w(width)
             .h_full()
+            .font("Segoe Fluent Icons")
```

Comment:
I moved the font configuration down to this component, as it makes more sense to be as close to where we render the icons as possible.